src/app/progressbar/progressbar.sample.ts

```diff
@@ -25,4 +28,9 @@ export class ProgressbarSampleComponent {
     randomIntFromInterval(min: number, max: number) {
         return Math.floor(Math.random() * (max - min + 1) + min);
     }
+
+    public ngOnInit() {
+        this.positionCenter = IgxTextAlign.CENTER;
```

Comment:
You can initialize the properties directly in the declaration. No need to do it `onInit`.